cmd/applyconfig/applyconfig.go

```diff
@@ -459,30 +458,13 @@ func fileFilter(info os.FileInfo, path string) (bool, error) {
 	return false, nil
 }
 
-type secretGetter struct {
-	sync.RWMutex
-	secrets sets.String
-}
-
-func (g *secretGetter) addSecrets(newSecrets ...string) {
-	g.Lock()
-	defer g.Unlock()
-	g.secrets.Insert(newSecrets...)
-}
-
-func (g *secretGetter) getSecrets() sets.String {
-	g.RLock()
-	defer g.RUnlock()
-	return g.secrets
-}
-
 var (
-	secrets *secretGetter
+	censor secrets.DynamicCensor
 )
 
 func init() {
-	secrets = &secretGetter{secrets: sets.NewString()}
-	logrus.SetFormatter(logrusutil.NewCensoringFormatter(logrus.StandardLogger().Formatter, secrets.getSecrets))
+	censor = secrets.NewDynamicCensor()
+	logrus.SetFormatter(censor.Formatter(logrus.StandardLogger().Formatter))
```

Comment:
Why are we making our own call to this? We do `LogrusInit` pretty much everywhere else, and it's applicable here, too.